### PR TITLE
some bugs fix include(autoCellHeight,multi grid drag,drag between grid two placeholder)

### PR DESCRIPTION
**Pull request by eagoo**
1.auto cellHeight fix
2.tow gridstack acceptWidgets bugs fix
3.widget can not resize after drag between two grid bug fix
4.drag between tow grid,two placeholder fix

### CODE REVIEW

**Comment by ghominejad** [comment]
Thank you

**Comment by radiolips** [comment]
I've been told that while this PR does fix the issues listed, it causes some other ones. Could anyone elaborate and/or set me straight? A couple parts seem a little iffy in the code, so an explanation would also be great.

File: dist/gridstack.js [hunk 1]
```diff
@@ -800,6 +800,9 @@
                         .removeData('draggable')
                         .removeClass('ui-draggable ui-draggable-dragging ui-draggable-disabled')
                         .unbind('drag', onDrag);
+                    if(el.find('div.ui-resizable-handle').length >0){
+                        el.find('div.ui-resizable-handle').remove()
```
Comment (adumesny) [inline dist/gridstack.js]: this for me fixes the issue where dragging a griditem to a second grid causes double style preventing resizing of it afterward.

**Comment by efx** [comment]
My team has verified the changes are needed to solve multiple grid issues like those mentioned in #718 . @eagoo could you please submit only the `src/gridstack.js` changes to the `devel` branch?

**Comment by radiolips** [comment]
Also, please make sure that it passes all tests, including jscs. I see a couple spots in the code that won't match the style, but they're easy fixes. Simply run `npm run build` locally and you'll see those results.

**Comment by adumesny** [comment]
note that "#4.drag between tow grid,two placeholder fix" doesn't appear anymore in the latest 1.0.0-dev code (using two.html sample code) so **not** all those fixes are needed.
the only one I see and commented on the fix is #718 which is a 2 liner fix... I recommend we pick that (I'll be happy to provide a PR)
"#1.auto cellHeight fix" I don't know what that issue is...

**Comment by radiolips** [comment]
A few of these have been individually merged. @eagoo , could you split out the remaining fixes into individual PRs and provide jsfiddles or explanations of how to arrive at the issue / solution? Thanks so much!

**Comment by ADobrianskiy** [comment]
Hey, guys. Do you have any updates about the issue with duplication? It still reproduces on the latest version here: http://gridstackjs.com/demo/two.html 
screencast: https://gfycat.com/evilspecificadder

**Pull request closed without merging**